AnnoyancesFilter/Cookies/sections/cookies_general.txt

```diff
@@ -11,6 +11,7 @@
 !#######################################################
 ! SECTION: Cookies - General element hiding
 !
+##.cookie_warning_overlay:remove()
```

Comment:
`:remove()` is not needed there. Remove it, add a domain to the rule, move the rule to cookies_specific.txt.